modules/oe_content_person/oe_content_person.module

```diff
@@ -156,30 +145,55 @@ function _oe_content_person_oe_person_jobs_validate(array &$element, FormStateIn
   // changed.
   foreach (Element::children($complete_form['oe_person_jobs']['widget']['entities']) as $key) {
     $item = $complete_form['oe_person_jobs']['widget']['entities'][$key];
-    if (!empty($item['#entity']) && !_oe_content_person_validate_person_job_entity($item['#entity'], $type)) {
+    if (empty($item['#entity'])) {
+      continue;
+    }
+    if ($type === 'eu') {
+      if ($item['#entity']->get('oe_role_reference')->isEmpty() && $item['#entity']->get('oe_role_name')->isEmpty()) {
```

Comment:
Just a suggestion, but maybe it would be more readable to assign these isEmpty checks to a variable like `$is_role_reference_empty`, or something similar.